.travis.yml

```diff
@@ -10,6 +10,8 @@ env:
     - SOLIDUS_BRANCH=v1.3 DB=postgresql
     - SOLIDUS_BRANCH=v1.4 DB=mysql
     - SOLIDUS_BRANCH=v1.4 DB=postgresql
+    - SOLIDUS_BRANCH=v2.1 DB=mysql
```

Comment:
Agreed and done - apologies on the mess here, I'm not super familiar with git or contributing in general yet. Thanks for bearing with me. 

Hopefully the last thing to solve is the strange testing error. I'll work on that and get something pushed either today or tomrrow.